test/test_filters.rb

```diff
@@ -463,7 +463,13 @@ def to_liquid
               g["items"].is_a?(Array),
               "The list of grouped items for 'default' is not an Array."
             )
-            assert_equal 5, g["items"].size
+            hr = "<---==========================================================--->"
```

Comment:
so that I could differentiate between various grouped items in the log, and hence count them easily..
That will be discarded in the final squash commit.